codec-mqtt/src/test/java/io/netty/handler/codec/mqtt/MqttCodecTest.java

```diff
@@ -69,6 +71,8 @@
     @Mock
     private final Attribute<MqttVersion> versionAttrMock = mock(Attribute.class);
 
+    private List<Object> out;
```

Comment:
- Could you release everything added here after each test, so the buffers don't leak?
- How about using an `ArrayList` or `ArrayDeque` instead of `LinkedList`? 
- How about making this a final field and let `tearDown()` clears the list?

Reply:
@vietj please do this and ping me once done

Reply:
I'll have a look soon

Reply:
LinkedList is what was previously used, I copied it, I'll change it to ArrayList instead

Reply:
ping @normanmaurer - test updated